lib/html-composer.js

```diff
@@ -94,6 +96,12 @@ function applyOptions(options, res) {
 
     // can't assign to result, or we'd get a circular reference in refs.
     _.set(state, '_components', componentList);
+    _.set(state, '_componentSchemas', _.map(componentList, function (name) {
```

Comment:
Eh, I think this is better because they're not just an array of path strings (but rather objects with some properties)